connectors/sources/jira.py

```diff
@@ -406,7 +408,6 @@ def get_default_configuration(cls):
             },
             "use_document_level_security": {
                 "display": "toggle",
-                "depends_on": [{"field": "data_source", "value": JIRA_CLOUD}],
                 "label": "Enable document level security",
                 "order": 14,
                 "tooltip": "Document level security ensures identities and permissions set in Jira are maintained in Elasticsearch. This enables you to restrict and personalize read-access users and groups have to documents in this index. Access control syncs ensure this metadata is kept up to date in your Elasticsearch documents.",
```

Comment:
Could you please add the information that you can only fetch 1000 users here?

Reply:
Added 👍